### PR TITLE
Create faul

### PR DESCRIPTION
**Pull request by KhongPhaiHoangQuangHuy**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by triach-rold — COMMENTED** [review]
404 error.

**Review by MaskDuck — COMMENTED** [review]
1. I guess the source of the code of the code is in https://github.com/KhongPhaiHoangQuangHuy/hoangquanghuy ? then the page will be able in https://faul.is-a.dev/hoangquanghuy and not the root one.
2. the file is supposed to be in the `domains` folder.
side thing: I agree with this statement
image

File: faul.json
Comment (CuteDog5695) [inline faul.json:1]: json file needs to be in domain folder

**Review by iostpa — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]
I was drunk while making my initial review, ignore it :)

**Pull request merged**